docs/workflow_solutions/creating_sandbox_apps.md

```diff
@@ -0,0 +1,180 @@
+# Open OnDemand
+
+[Open OnDemand](https://rc.uab.edu)(OOD) is the web interface to Cheaha HPC resources. We have setup multiple general purpose apps, such as HPC Desktop, Jupyter, RStudio etc, on Cheaha, to make the process of accessing these basic softwares easier. OOD also allows users to setup their own personal applications, as Sandbox apps.
+
+This article will walk you through how to set up an OOD Sandbox app. For ease of understanding, the article is split into the following sections:
+
+1. [Setting up OOD Sandbox for your Cheaha Account](#setting-up-ood-sandbox-for-your-cheaha-account)
+1. [Building your first dev app](#building-your-first-dev-app)
+   - [FSL](#fsl)
+1. [Launching your OOD Sandbox App](#launching-your-ood-sandbox-app)
+1. [More info](#more-info)
+
+## Setting up OOD Sandbox for your Cheaha Account
+
+The following steps will guide you to set up Sandbox for your Cheaha account.
+
+<!-- markdownlint-disable MD046 -->
+!!! note
+
+    This is a one time setup, and would not need to be repeated for subsequent apps you would like to create.
+<!-- markdownlint-enable MD046 -->
+
+Create a dev folder in your `$USER/ondemand` folder by following these steps:
+
+1. Create an HPC Desktop Interactive Job on Cheaha. There is a detailed guide [here](../cheaha/open_ondemand/ood_layout.md#creating-an-interactive-job) on how to do this.
+
+1. Access the terminal within your HPC Desktop Job. There is a guide on how to access the terminal [here](../cheaha/tutorial/pytorch_tensorflow.md#installing-anaconda-environments-using-the-terminal)
+
+    ![Terminal button from Interactive sessions page on Cheaha](images/cheaha_sandbox_shell_button.png)
+
+1. Create the folder within the terminal using the command `mkdir -p $USER_DATA/ondemand/dev`.
+
+1. Go to the [Cheaha](https://rc.uab.edu) Open OnDemand homepage.
+
+1. When logged in, click on the "Help" button, you should see the button in the top-right corner of the landing page of Cheaha, click on "Restart Web Server".
+
+    ![Help button menu options](images/cheaha_help_button.png)
+
+1. When the page reloads, you should see a new "Develop" tab.
+
+    ![Develop button on Cheaha](images/cheaha_develop_button.png)
+
+## Building Your First Dev App
+
+Now that Sandbox is set up on your account, we can create an App. The instructions below detail how to create your first example dev app via a container:
```

Comment:
The implication of a subsection is that there are, or will be, more than one. Can you rework the intro to account for that? Right now the intro ends with the assumption the reader will go directly into the subsection. This may be true, but may be less accessible and less future proof than a self-contained intro that explicitly guides the reader where to go next.

Reply:
We show the OOD file editor in the screenshots, which is fine. You may want to let people know that, if they wish to develop custom applications, they may find it more efficient to use an IDE or something like VSCode.